java/piranha/build.gradle

```diff
@@ -30,6 +30,7 @@ dependencies {
     annotationProcessor deps.apt.autoService
 
     compileOnly deps.build.errorProneCheckApi
+    implementation deps.build.googleJsonSimple
```

Comment:
I'm new to gradle. Can you double check that `implementation` is the correct thing to use here?

Reply:
Oh and don't forget this one!